src/main/java/com/spotify/sparkey/Util.java

```diff
@@ -270,4 +270,32 @@ static void nonThrowingClose(Closeable stream) {
       e.printStackTrace();
     }
   }
+
+  static boolean renameFile(File srcFile, File destFile) {
+    if (!destFile.exists()) {
```

Comment:
Since this is common function now (even if these are implicitly handled), it might be good idea to do early entry checks on: 
- srcFile exists 
- srcFile n destFile are same.